libr/util/file.c

```diff
@@ -211,11 +211,18 @@ R_API char *r_file_abspath(const char *file) {
 			return strdup (file);
 		}
 		if (!strchr (file, ':')) {
-			char *abspath = malloc (MAX_PATH);
+			PTCHAR abspath = malloc (MAX_PATH * sizeof (TCHAR));
 			if (abspath) {
-				GetFullPathName (file, MAX_PATH, abspath, NULL);
-				ret = strdup (abspath);
+				PTCHAR f = r_sys_conv_utf8_to_utf16 (file);
```

Comment:
Currently radare2 can be built with UNICODE and without, please add corresponding checks, if it is defined then do conv